### DISCUSSION TITLE
Carbon Black Flagged DBATools for Malicious Content

### DISCUSSION THREAD

**Discussion by avargas19** [dataplat/dbatools#8417] [Q&A]
I have tried in several computers/servers, uninstalled and reinstalled the module. Talked to cyber security try to get policy exception and they are concerned that something is off. Anyone else with the same problem? Any ideas or pointers? Thanks.

PS C:\Users\avargas2> Import-Module dbatools
At C:\Users\avargas2\Documents\WindowsPowerShell\Modules\dbatools\1.1.106\allcommands.ps1:1 char:1
+ ### DO NOT EDIT THIS FILE DIRECTLY ###
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
This script contains malicious content and has been blocked by your antivirus software.
    + CategoryInfo          : ParserError: (:) [], ParseException
    + FullyQualifiedErrorId : ScriptContainedMaliciousContent

**Comment by lowlydba** [comment]
I think this great article still holds mostly true and may be a good starting point: [talking to your security team about powershell and dbatools ](https://dbatools.io/secure/)
